src/absil/ilread.fs

```diff
@@ -3970,15 +3970,16 @@ let ClosePdbReader pdb =
 #endif
 
 let OpenILModuleReader infile opts = 
-    if IL.runningOnWindows then
```

Comment:
Thanks for opening the [#1094](https://github.com/Microsoft/visualfsharp/issues/1094) issue, I incorrectly assumed it was using the bcl underneath.